<?xml version="1.0"?>
<!-- This file can be launched by start_robot.launch or by start_simulation.launch -->
<!-- Do not launch manually! -->

<launch>
  <arg name="robot_name" default="omega" />
  <arg name="ground_truth" default="False" />
  <arg name="debug" default="True" />
  
  <group ns="$(arg robot_name)">
    <!-- define debug config -->
    <param name="debugger/active" type="bool" value="$(arg debug)" />
    <!-- start omega node -->
    <node name="omega_node" pkg="omega" type="omega" output="screen" respawn="true" />
  </group>

  <!-- start debug visualization -->
  <node if="$(arg debug)" name="image_view_node" pkg="image_view" type="image_view" output="screen" respawn="true" >
    <param name="autosize" type="bool" value="true" />
    <remap from="image" to="/$(arg robot_name)/debugger/image" />
  </node>
</launch>
